src/model/destination-model.js

```diff
@@ -0,0 +1,18 @@
+export default class ModelDestination{
```

Comment:
Класс должен называться так же как и файл, только в другой нотации
```suggestion
export default class DestinationModel {
```
Можно так же называть во множественном числе, но это не критично, например DestinationsModel. Но все зависит от модели и их паттерна, иногда нужно называть только в единственном (не в этом проекте)